### ISSUE TITLE
[Multi-Storage] Queue issues (Item loss, Stack mode not working)

### ISSUE THREAD

**Issue by jezzen** [ShetiPhian/Localizations-and-Issues#173]
This is on Multi-Storage 1.4.9 in the Modern Skyblock 3 pack.

TLDR: Dense Queue stack mode not working, Dense Queues consuming 2 items per "transfer". Normal Queues work fine

I was looking around for good vertical item transfer in gated mode prior to having access to any kind of transfer pipe and found the queue from multi-storage. It could be configured to accept/output items from any sides and the dense version could manage stacks. Although i found that it required a redstone signal , that could be worked around with redstone clocks.

However, even with the dropper: stack mode set, it only output a single item at a time. And after noticing the number of test items reduced, I found the queue to be consuming 2 items (in all dropper modes, start and end, stack and single) per activation.

Further testing with another normal queue it worked just fine (yay, single item stacks).

**Comment by ShetiPhian** [comment]
Finally got a sliver of free time;

This build looks to fix it:
http://www.mediafire.com/file/ok3rdcjaw5t8733/multistorage-1.12.0-1.4.10-TESTER.jar

**Comment by ShetiPhian** [comment]
New build uploaded to Curse
https://minecraft.curseforge.com/projects/multistorage/files/2556834
